### DISCUSSION TITLE
dm 은 어떤 화면에서 이루어 지나요?

### DISCUSSION THREAD

**Discussion by gangukss** [42-Tomodachi/ft_transcendence#30] [Q&A]
피그마에 dm 보내기가 있고 서브젝트에도 로그인 한 유저는 상담할 수 있어야 한다 라고 되어있습니다. 
dm 어떻게 구현하면 될까요?

**Comment by som-syom** [comment]
dm 도 채팅방의 종류중 하나라고 생각하고 제작했습니다. figma 상에서 유저리스트에서 유저를 선택하고 나온 모달에서 DM보내는 버튼을 누르면 1:1채팅방이 생성되면서 바로 입장 되도록 하는 흐름으로 생각했습니다.. 저희가 이런 요청을 백엔드로 보내면 거기서 새로운 채팅방을 생성하고, 유저1,2의 참여중 채팅방 리스트에도 추가되도록 하면 되지않을까 싶은데 ... 더 고민해봐야할 부분이 있을까요?

**Reply by gangukss** [reply]
'~의 채팅방' 화면만 있어서 궁금해서 여쭤봤습니다. 질문이 다시보니까 좀 애매하네요.. 그럼 같은 화면에서 dm도 진행되는 것으로 이해 할게요!
